### PR TITLE
Updated mention of services in top text to reflect new service names

### PR DESCRIPTION
**Pull request by atxbyea**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/vacuum.xiaomi_miio.markdown [hunk 1]
```diff
@@ -20,7 +20,11 @@ Currently supported services are:
 - `clean_spot`
 - `set_fan_speed`
 - remote control of your robot.
-- `xiaomi_clean_zone`
+- `xiaomi_miio.vacuum_clean_zone`
```
Comment (frenck) [inline source/_integrations/vacuum.xiaomi_miio.markdown:23]: Why are these prefixed by the domain and the ones already in the docs not?

**Review by frenck — CHANGES_REQUESTED** [review]
Left a small question, @atxbyea, could you help me answer that?

**Comment by frenck** [comment]
@atxbyea Ping! 😄

**Comment by atxbyea** [comment]
good question @frenck I just saw that list in the top of the docs, and saw that it referenced the "old" service name, and changed it to the "new" service name, probably it could just say `clean_zone` to be in line with the rest of them ?

**Comment by frenck** [comment]
@atxbyea There is no policy on that, however, I think we should be consistent on the same page at least 😉

**Comment by frenck** [comment]
Closing PR since it is running stale.

**Pull request closed without merging**